components/validation/src/main/java/org/wso2/carbon/identity/x509Certificate/validation/CertificateValidationUtil.java

```diff
@@ -460,100 +487,155 @@ private static String getCACertsRegPath(X509Certificate peerCertificate) throws
                 replaceAll("%", ":");
     }
 
-    private static List<CACertificate> getCACertsFromRegResource(String caRegPath) throws RegistryException,
-            CertificateValidationException {
-
-        List<CACertificate> caCertificateList = new ArrayList<>();
-        //get tenant registry for loading validator configurations
-        Registry registry = getGovernanceRegistry(
-                PrivilegedCarbonContext.getThreadLocalCarbonContext().getTenantDomain());
-
-        if (registry.resourceExists(caRegPath)) {
-            Collection collection = (Collection) registry.get(caRegPath);
-            if (collection != null) {
-                String[] children = collection.getChildren();
-                for (String child : children) {
-                    Resource resource = registry.get(child);
-                    CACertificate caCertificate = resourceToCACertObject(resource);
-                    caCertificateList.add(caCertificate);
+    private static List<CACertificate> getCACertsFromRegResource(String issuerDN) throws
```

Comment:
Since we are not getting this from registry, we can rename the method.

Reply:
https://github.com/wso2-extensions/identity-x509-commons/pull/47/commits/db7982fd5d7c401c9a801a720600815dfd8d0f38